src/coremotion.cs

```diff
@@ -561,7 +561,10 @@ interface CMMovementDisorderManager {
 		[Export ("queryTremorFromDate:toDate:withHandler:")]
 		void QueryTremor (NSDate fromDate, NSDate toDate, CMTremorResultHandler handler);
 
-		[Export ("lastProcessedDate")]
+		[NullAllowed, Export ("lastProcessedDate")]
 		NSDate LastProcessedDate { get; }
+
+		[NullAllowed, Export ("monitorKinesiasExpirationDate")]
+		NSDate MonitorKinesiasExpirationDate { get; }
```

Comment:
Strange there is no min version on this but matches header.

Reply:
yeah new type :)

```csharp
[Watch (5,0), NoTV, NoMac, NoiOS]
[BaseType (typeof (NSObject))]
interface CMMovementDisorderManager {
```